### DISCUSSION TITLE
xdrip plus "sensor stopped" + "calibration rejected" messages

### DISCUSSION THREAD

**Discussion by PeterCGM** [NightscoutFoundation/xDrip#2374] [Q&A]
Hi All,

Yesterday I started a G5 transmitter with replaced batteries and just in case did "hard reset transmitter" which seem to work fine. Now xdrip plus started to also work fine, measurements are correct as usual. I am only asking myself are those messages "sensor stopped" and "calibration rejected" for real, should I do something about that or just leave them be?
![Screenshot_20221009-075014](https://user-images.githubusercontent.com/71789897/194739516-36ca8f65-53ab-416e-a0cb-3f5f64115b0e.png)
![Screenshot_20221009-074933](https://user-images.githubusercontent.com/71789897/194739517-4db6c9b5-c622-42ce-a460-55a574ef7439.png)

**Comment by Navid200** [comment]
If you have readings, there should be no messages suggesting that the sensor is stopped after a few read cycles (5 minutes each).
Do you still see the message?

Since you are running G5 in native mode, you are calibrating the transmitter, not xDrip.
If the transmitter rejects a calibration, there is nothing xDrip can do about it.

I am not clear exactly what you are experiencing considering you have only two screenshots.  For all I know, those may be showing things that are not there any longer.

**Reply by PeterCGM** [reply]
In fact those notifications are always there, but fine I will just ignore them

**Reply by Navid200** [reply]
Can you post a screenshot of the notification?

**Reply by PeterCGM** [reply]
I triggered 3 x "hard reset transmitter" and the error disappeared
